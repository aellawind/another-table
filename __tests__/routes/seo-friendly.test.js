const restaurants = require('../../src/data/restaurants').restaurants;
const { int2kebab, kebab2int } = require('../../src/routes/seo-friendly');

describe('int2kebab & kebab2int', () => {
  restaurants.forEach(({ name, id }) => {
    test(name, () => {
      const kebab = int2kebab(id);
      expect(kebab).toMatchSnapshot();
      const int = kebab2int(kebab);
      expect(int).toEqual(id);
    });
  });
});
