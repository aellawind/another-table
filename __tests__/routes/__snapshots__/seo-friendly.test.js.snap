// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`int2kebab & kebab2int 1300 on Fillmore 1`] = `"1300-on-fillmore"`;

exports[`int2kebab & kebab2int Absinthe Brasserie and Bar 1`] = `"absinthe-brasserie-and-bar"`;

exports[`int2kebab & kebab2int Ala Romana 1`] = `"ala-romana"`;

exports[`int2kebab & kebab2int Amber India 1`] = `"amber-india"`;

exports[`int2kebab & kebab2int Ame 1`] = `"ame"`;

exports[`int2kebab & kebab2int Aslam's Rasoi 1`] = `"aslam's-rasoi"`;

exports[`int2kebab & kebab2int Aziza 1`] = `"aziza"`;

exports[`int2kebab & kebab2int B44 1`] = `"b44"`;

exports[`int2kebab & kebab2int BIN 38 1`] = `"bin-38"`;

exports[`int2kebab & kebab2int Bambino's Ristorante 1`] = `"bambino's-ristorante"`;

exports[`int2kebab & kebab2int Beach Chalet Brewery & Restaurant 1`] = `"beach-chalet-brewery-&-restaurant"`;

exports[`int2kebab & kebab2int Belcampo 1`] = `"belcampo"`;

exports[`int2kebab & kebab2int Bella Trattoria Italiana 1`] = `"bella-trattoria-italiana"`;

exports[`int2kebab & kebab2int Benu 1`] = `"benu"`;

exports[`int2kebab & kebab2int Bistro Boudin 1`] = `"bistro-boudin"`;

exports[`int2kebab & kebab2int Blue Plate 1`] = `"blue-plate"`;

exports[`int2kebab & kebab2int Bob's Steak & Chop House - San Francisco 1`] = `"bob's-steak-&-chop-house---san-francisco"`;

exports[`int2kebab & kebab2int Bouche 1`] = `"bouche"`;

exports[`int2kebab & kebab2int Boulevard 1`] = `"boulevard"`;

exports[`int2kebab & kebab2int Boxing Room 1`] = `"boxing-room"`;

exports[`int2kebab & kebab2int Brandy Ho's Hunan Food 1`] = `"brandy-ho's-hunan-food"`;

exports[`int2kebab & kebab2int Cafe Claude - Downtown 1`] = `"cafe-claude---downtown"`;

exports[`int2kebab & kebab2int Cafe Tiramisu 1`] = `"cafe-tiramisu"`;

exports[`int2kebab & kebab2int Cafe Zoetrope 1`] = `"cafe-zoetrope"`;

exports[`int2kebab & kebab2int Caffe Sport - San Francisco 1`] = `"caffe-sport---san-francisco"`;

exports[`int2kebab & kebab2int Cajun Pacific 1`] = `"cajun-pacific"`;

exports[`int2kebab & kebab2int Campton Place 1`] = `"campton-place"`;

exports[`int2kebab & kebab2int Capannina 1`] = `"capannina"`;

exports[`int2kebab & kebab2int Capurro's 1`] = `"capurro's"`;

exports[`int2kebab & kebab2int Castagna - San Francisco 1`] = `"castagna---san-francisco"`;

exports[`int2kebab & kebab2int Causwells 1`] = `"causwells"`;

exports[`int2kebab & kebab2int Claudine 1`] = `"claudine"`;

exports[`int2kebab & kebab2int Craw Station 1`] = `"craw-station"`;

exports[`int2kebab & kebab2int Dirty Habit 1`] = `"dirty-habit"`;

exports[`int2kebab & kebab2int Espetus Churrascaria - San Francisco 1`] = `"espetus-churrascaria---san-francisco"`;

exports[`int2kebab & kebab2int Farallon 1`] = `"farallon"`;

exports[`int2kebab & kebab2int Fishermen's Grotto #9 1`] = `"fishermen's-grotto-#9"`;

exports[`int2kebab & kebab2int Fondue Cowboy 1`] = `"fondue-cowboy"`;

exports[`int2kebab & kebab2int Foreign Cinema 1`] = `"foreign-cinema"`;

exports[`int2kebab & kebab2int Garcon 1`] = `"garcon"`;

exports[`int2kebab & kebab2int Hakkasan San Francisco 1`] = `"hakkasan-san-francisco"`;

exports[`int2kebab & kebab2int Ichi Sushi & Ni Bar 1`] = `"ichi-sushi-&-ni-bar"`;

exports[`int2kebab & kebab2int Ideale 1`] = `"ideale"`;

exports[`int2kebab & kebab2int Il Borgo 1`] = `"il-borgo"`;

exports[`int2kebab & kebab2int Infusion Lounge 1`] = `"infusion-lounge"`;

exports[`int2kebab & kebab2int Isa 1`] = `"isa"`;

exports[`int2kebab & kebab2int Jai Yun 1`] = `"jai-yun"`;

exports[`int2kebab & kebab2int Jeanne D'Arc - Cornell Hotel De France 1`] = `"jeanne-d'arc---cornell-hotel-de-france"`;

exports[`int2kebab & kebab2int KUSAKABE 1`] = `"kusakabe"`;

exports[`int2kebab & kebab2int La Nebbia Restaurant 1`] = `"la-nebbia-restaurant"`;

exports[`int2kebab & kebab2int Lark Creek Steak 1`] = `"lark-creek-steak"`;

exports[`int2kebab & kebab2int Le Central 1`] = `"le-central"`;

exports[`int2kebab & kebab2int Le Charm 1`] = `"le-charm"`;

exports[`int2kebab & kebab2int Le Zinc 1`] = `"le-zinc"`;

exports[`int2kebab & kebab2int Limon Rotisserie â€“ Bay View 1`] = `"limon-rotisserie-â€“-bay-view"`;

exports[`int2kebab & kebab2int Mamacita 1`] = `"mamacita"`;

exports[`int2kebab & kebab2int Mangia Tutti 1`] = `"mangia-tutti"`;

exports[`int2kebab & kebab2int Maruya 1`] = `"maruya"`;

exports[`int2kebab & kebab2int Maykadeh 1`] = `"maykadeh"`;

exports[`int2kebab & kebab2int Mazza Luna 1`] = `"mazza-luna"`;

exports[`int2kebab & kebab2int Minas Brazilian Restaurant 1`] = `"minas-brazilian-restaurant"`;

exports[`int2kebab & kebab2int NEPTUNE's Waterfront Grill & Bar 1`] = `"neptune's-waterfront-grill-&-bar"`;

exports[`int2kebab & kebab2int New Delhi Restaurant 1`] = `"new-delhi-restaurant"`;

exports[`int2kebab & kebab2int North Beach Restaurant 1`] = `"north-beach-restaurant"`;

exports[`int2kebab & kebab2int Opaque - Dining In The Dark 1`] = `"opaque---dining-in-the-dark"`;

exports[`int2kebab & kebab2int Outerlands 1`] = `"outerlands"`;

exports[`int2kebab & kebab2int Panta Rei 1`] = `"panta-rei"`;

exports[`int2kebab & kebab2int Perry's Union Street 1`] = `"perry's-union-street"`;

exports[`int2kebab & kebab2int Piccino 1`] = `"piccino"`;

exports[`int2kebab & kebab2int Players Sports Grill & Arcade 1`] = `"players-sports-grill-&-arcade"`;

exports[`int2kebab & kebab2int Rich Table 1`] = `"rich-table"`;

exports[`int2kebab & kebab2int Ristobar 1`] = `"ristobar"`;

exports[`int2kebab & kebab2int Rocco's Cafe 1`] = `"rocco's-cafe"`;

exports[`int2kebab & kebab2int Sauce: Gough 1`] = `"sauce:-gough"`;

exports[`int2kebab & kebab2int Serpentine 1`] = `"serpentine"`;

exports[`int2kebab & kebab2int Skool 1`] = `"skool"`;

exports[`int2kebab & kebab2int Sons & Daughters 1`] = `"sons-&-daughters"`;

exports[`int2kebab & kebab2int South at SF Jazz 1`] = `"south-at-sf-jazz"`;

exports[`int2kebab & kebab2int Spruce 1`] = `"spruce"`;

exports[`int2kebab & kebab2int Star India 1`] = `"star-india"`;

exports[`int2kebab & kebab2int Stones Throw 1`] = `"stones-throw"`;

exports[`int2kebab & kebab2int Sutro's at the Cliff House 1`] = `"sutro's-at-the-cliff-house"`;

exports[`int2kebab & kebab2int TBD Restaurant 1`] = `"tbd-restaurant"`;

exports[`int2kebab & kebab2int Tal-Y-Tara Tea & Polo Shoppe 1`] = `"tal-y-tara-tea-&-polo-shoppe"`;

exports[`int2kebab & kebab2int TamashiSoul Sushi Bar 1`] = `"tamashisoul-sushi-bar"`;

exports[`int2kebab & kebab2int The Oak Room - The Westin St. Francis 1`] = `"the-oak-room---the-westin-st.-francis"`;

exports[`int2kebab & kebab2int The Sausage Factory 1`] = `"the-sausage-factory"`;

exports[`int2kebab & kebab2int Thirsty Bear 1`] = `"thirsty-bear"`;

exports[`int2kebab & kebab2int Tres - SF (fka Tres Agaves) 1`] = `"tres---sf-(fka-tres-agaves)"`;

exports[`int2kebab & kebab2int TropisueÃ±o 1`] = `"tropisueã±o"`;

exports[`int2kebab & kebab2int Trou Normand 1`] = `"trou-normand"`;

exports[`int2kebab & kebab2int Urban Putt 1`] = `"urban-putt"`;

exports[`int2kebab & kebab2int Vicoletto 1`] = `"vicoletto"`;

exports[`int2kebab & kebab2int Waterbar 1`] = `"waterbar"`;

exports[`int2kebab & kebab2int Woodwards Garden 1`] = `"woodwards-garden"`;

exports[`int2kebab & kebab2int YamaSho 1`] = `"yamasho"`;

exports[`int2kebab & kebab2int Yank Sing - Rincon Center 1`] = `"yank-sing---rincon-center"`;

exports[`int2kebab & kebab2int Yank Sing - Stevenson Street 1`] = `"yank-sing---stevenson-street"`;

exports[`int2kebab & kebab2int cityhouse Restaurant - Parc 55 Hotel 1`] = `"cityhouse-restaurant---parc-55-hotel"`;

exports[`int2kebab & kebab2int mrCrave / bistro + cafe 1`] = `"mrcrave-/-bistro-+-cafe"`;
